hexa/workspaces/graphql/schema.graphql

```diff
@@ -177,7 +177,9 @@ enum WorkspaceMembershipRole {
   VIEWER  # The user can view the workspace and its contents.
 }
 
-
+type WorkspaceCandidatesResult {
+    items: [User!]!
```

Comment:
If you do not do a pagination, let remove this type and just return a list of User